vitest.config.ts

```diff
@@ -0,0 +1,9 @@
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    include: ['__test__/**.spec.ts'],
```

Comment:
the default value is `**/*.{test,spec}.?(c|m)[jt]s?(x)` so you can remove this option

Reply:
No, this option is used to limit to only do test in root folder